tests/integration/gql/test_ensembles.py

```diff
@@ -7,14 +7,14 @@
   ensemble(id: $id) {
     id
     size
-    inputs
+    parameterNames
```

Comment:
Why camel-casing here?

Reply:
GraphQL changes automatically underscore to camel case. `parameter_names` becomes `parameterNames`

Reply:
Yep. GraphQL is a Javascript library originally, where the style is camelcase. Python's graphene automatically converts the names for you.